src/shared/test-utils/index.js

```diff
@@ -31,8 +32,14 @@ const AppTree = ({ children }) => (
     </AppConfigProvider>
 );
 
-AppTree.propTypes = {
+Providers.propTypes = {
     children: PropTypes.element.isRequired,
 };
 
-export default AppTree;
+const customRender = (ui, options) => render(ui, { wrapper: Providers, ...options });
```

Comment:
We might want to do this for next with moxy as well!

Reply:
Yep! Personally, I prefer this approach which is actually pretty much a shameless copy from the docs, but it works well! 👌 

- RTL: https://testing-library.com/docs/react-testing-library/setup#custom-render
- NTL: https://www.native-testing-library.com/docs/setup#custom-render

I'll notify the team in Slack to check who's available to port this over to Next.js with MOXY.